yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -78,6 +79,12 @@ func (yb *TargetYugabyteDB) Exec(query string) (int64, error) {
 
 	res, err := yb.db.Exec(query)
 	if err != nil {
+		var pgErr *pgconn5.PgError
+		if errors.As(err, &pgErr) {
+			if pgErr.Hint != "" || pgErr.Detail != "" {
+				return rowsAffected, fmt.Errorf("run query %q on target %q: %w HINT:%q DETAIL:%q", query, yb.tconf.Host, err, pgErr.Hint, pgErr.Detail)
```

Comment:
nit: next lines for HINT and DETAIL - `fmt.Errorf("run query %q on target %q: %w\n HINT:%q\n DETAIL:%q", query, yb.tconf.Host, err, pgErr.Hint, pgErr.Detail)`